Dockerfile

```diff
@@ -18,6 +18,8 @@ RUN apk add --no-cache \
 		py-setuptools=33.1.1-r1 \
 		py-yaml=3.12-r1 \
 		tar=1.29-r1 \
+		openssh=7.5_p1-r8 \
```

Comment:
Is this dependency required?  Looking at the alpine package repo, it is not listed as a dependency for sshpass.    https://pkgs.alpinelinux.org/package/v3.7/main/x86/sshpass

Reply:
ok